src/EnergyPlus/HeatRecovery.cc

```diff
@@ -529,6 +532,12 @@ namespace HeatRecovery {
                                  "\" latent heating effectiveness at 75% rated flow is less than at 100% rated flow.");
                 ShowContinueError("Latent heating effectiveness at 75% rated flow is usually greater than at 100% rated flow.");
             }
+            if ((ExchCond(ExchNum).HeatEffectLatent100 == 0.0 && ExchCond(ExchNum).HeatEffectLatent75 > 0.0) || (ExchCond(ExchNum).HeatEffectLatent100 > 0.0 && ExchCond(ExchNum).HeatEffectLatent75 == 0.0)) {
+                ShowWarningError( cCurrentModuleObject + " \"" + ExchCond( ExchNum ).Name + "\"" );
+                ShowContinueError("..." + cNumericFieldNames(3)  + " = " + RoundSigDigits(rNumericArgs(3), 2));
+                ShowContinueError("..." + cNumericFieldNames(5)  + " = " + RoundSigDigits(rNumericArgs(5), 2));
+                ShowContinueError("...Latent heating effectiveness at 75% and 100% rated flow can be 0.0 but not one of them only.");
+            }
```

Comment:
Throws warning when only of the heating latent effectiveness (at 100% or 75% of rated flow) user-input values are zero.